_include/um/register/claim_admin.rst

```diff
@@ -0,0 +1,232 @@
+Claim Administrators Administration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+  The register contains employees of contractual health facilities responsible for preparation and/or submission of claims. Administration of the register of claim administrators is restricted to users with the role of Scheme Administrator.
+
+Pre-conditions
+""""""""""""""
+
+  A claim administrator may be added after the agreement of a contractual health facility and the management of the scheme administration.
+
+Navigation
+""""""""""
+
+  All functionality for use with the administration of claim administrators can be found under the main menu ``Administration``, submenu ``Claim Administrators``.
+
+  .. _image67:
+  .. figure:: /img/user_manual/image60.png
+    :align: center
+
+    `Image 67 - Navigation Claim Administrators`
+
+  Clicking on the sub menu ``Claim Administrators`` re-directs the current user to the `Claim Administrators Control Page. <#claim-administrators-control-page>`__
+
+  .. _image68:
+  .. figure:: /img/user_manual/image56.png
+    :align: center
+
+    `Image 68 - Claim Administrators Control Page`
+
+Claim Administrators Control Page
+"""""""""""""""""""""""""""""""""
+
+  The ``Claim Administrators Control Page`` is the central point for all claim administrators administration. By having access to this panel, it is possible to add, edit, delete and search claim administrators. The panel is divided into four panels (:ref:`Image 68<image68>`).
```

Comment:
something wrong here 'The ``Claim Administrators Control Page`` is the central point for all claim administrators administration'

administartors adminstration should be improved i suppose